src/charm.py

```diff
@@ -29,6 +38,7 @@ class MagmaAccessGatewayOperatorCharm(CharmBase):
     def __init__(self, *args):
         """Observes juju events."""
         super().__init__(*args)
+        self.orchestrator_requirer = OrchestratorRequires(self, "magma-orchestrator")
```

Comment:
Corresponding entry in `metadata.yaml` is missing.

Reply:
It was previously added by @gruyaume, so it is not part of this PR.